### ISSUE TITLE
Error after fresh installation

### ISSUE THREAD

**Issue by exocode** [tenKinetic/Knerve#5]
```
A JavaScript error occurred in the main process

Uncaught Exception:
TypeError: Cannot read property '0' of null
    at fs.readFile (/Applications/Knerve.app/Contents/Resources/app.asar/build/main.min.js:3:10244)
    at tryToString (fs.js:513:3)
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:501:12)
```

LOG

```
2018-07-18 15:47:58: ERROR: Error reading modified settings file: Error: ENOENT: no such file or directory, open '/Users/janjezek/Library/Application Support/Knerve/knerve.json'
2018-07-18 15:48:04: INFO: Loaded settings:
2018-07-18 15:48:04: INFO: {"mysql":{"enabled":false,"start":"/usr/local/mysql/bin/mysqld_safe --user=root","status":"/usr/local/mysql/bin/mysqladmin version","stop":"/usr/local/mysql/support-files/mysql.server stop"},"sites":[]}
2018-07-18 15:48:12: INFO: DNS server started with config: {"bin":"/Applications/Knerve.app/Contents/Resources/app.asar/build/bin/pow","dstPort":80,"httpPort":20559,"dnsPort":20560,"timeout":900,"workers":2,"domains":["knerve"],"extDomains":[],"hostRoot":"/Users/janjezek/Library/Application Support/Pow/Hosts","logRoot":"/Users/janjezek/Library/Logs/Pow","rvmPath":"/Users/janjezek/.rvm/scripts/rvm"}
2018-07-18 15:48:12: INFO: Bounce server running on secure port 61525
2018-07-18 15:48:12: INFO: Bounce server running on port 61526
2018-07-18 15:48:15: INFO: MySQL installation not detected


```

**Comment by tenKinetic** [comment]
I have published a new version ([1.1.1](https://github.com/tenKinetic/Knerve/releases/tag/v1.1.1)) which should handle this error.
The underlying problem will remain though which is that the packet filter command failed to acquire a token.
The new version may provide some info on why that is though.

**Comment by tenKinetic** [comment]
Okay, 1.1.2 will be uploaded shortly which does fix this for me.
